mozilla/activity-stream
# Bug 1546734 - [Card] Adjust vertical spacing and text color

Testing: 

- Run `dev-test-all`
- See Wolasi's [requested design tweaks](https://bugzilla.mozilla.org/show_bug.cgi?id=1546734) and verify they match
